packages/@glimmer/global-context/index.ts

```diff
@@ -100,6 +100,15 @@ export let setProp: (obj: object, prop: string, value: unknown) => void;
  */
 export let getPath: (obj: object, path: string) => unknown;
 
+/**
+ * Hook for specifying how Glimmer should update paths in cases where it needs
+ * to. For instance, when updating a template reference (e.g. 2-way-binding)
+ *
+ * @param obj The object provided to get a value from
+ * @param path The path to get the value from
+ */
+export let setPath: (obj: object, path: string, value: unknown) => unknown;
```

Comment:
This is specifically for `{{get}}`. In general, when we see a path in Glimmer, we break it down into a chain of references. However, when the path is dynamic, we cannot do that, we have to do it all within a single reference. We already had `getPath` because there was an instance of this before for getting a path (the `key` on `each`) but we never encountered a `setPath` before. `{{get}}` can also _update_ the value that is accessed (similar to how `mut` works), so that's why we need the ability to set a path - it's dynamic and not broken down into pieces.

Reply:
So to clarify, its for `(mut (get` being used together (since I didn't think that `{{something whatever=(get otherThing 'lol.path')}} allowed mutation of `otherThing.lol.path`)? Is that right?